### ISSUE TITLE
Block Physics disregards the settings made while using certain resource packs [BUG]

### ISSUE THREAD

**Issue by NaiNonTH** [haubna/PhysicsMod#626]
**Describe the bug**
Affected by Foliage+ and Foliage+ Tweaks resource packs, the Physics Mod doesn't adhere to the Block Physics settings made (except Particles) on some blocks — it instead works as Blocky block physics.

**To Reproduce**
1. Download and Install Foliage+ or Foliage+ Tweaks resource pack.
2. Launch Minecraft, then launch any world you'd like.
3. Set the Block Physics settings to anything except Off, Blocky, or Particles.
4. Try getting yourself some grass blocks or sand blocks and placing them.
5. Break them, and you'll see that Block Physics appears to be a Blocky type.

**Screenshots (Video)**
https://i.imgur.com/FVwYiyS.mp4

Minecraft Version: 1.19.4
Physics Mod Version: 2.12.1 (It was like this for a long time, but I just had the chance to report it.)

**Comment by NaiNonTH** [comment]
Links to resource packs mentioned:
Foliage+ – https://modrinth.com/resourcepack/foliage+
Foliage+ Tweaks – https://modrinth.com/resourcepack/foliage+-tweaks

**Comment by Muffintiem** [comment]
That sounds like what's happening with the grass blocks using Tooniverse. Same exact behavior.

**Comment by SirDinoswar** [comment]
Yeah Noticed that it also has a problem with Better Foliage Renewed, has both the grass and leaves issues.
